lib/ansible/modules/cloud/amazon/rds_cluster_param_group.py

```diff
@@ -0,0 +1,298 @@
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+module: rds_cluster_param_group
+short_description: Manages RDS Aurora Cluster Parameter Groups
+description:
+  -Manages the creation, modification, deletion of RDS Aurora parameter groups.
+version_added: "2.3"
+author: "Nick Aslanidis (@naslanidis)"
+options:
+  param_group_name:
+    description:
+      - The name of the DB cluster parameter group.
+    required: true
+    default: null
+  param_group_family:
+    description:
+      - The DB cluster parameter group family name.
+    required: false
+    default: null
+  description:
+    description:
+      - The description for the DB cluster parameter group.
+    required: false
+    default: null
+  params:
+    description:
+      - A list of parameters in the DB cluster parameter group
+    required: false
+    default: null
+  apply_method:
+    description:
+      - Indicates when to apply parameter updates.
+    required: false
+    default: null
+extends_documentation_fragment:
+  - aws
+  - ec2
+'''
+
+EXAMPLES = '''
+# Create RDS cluster param group with no parameters set
+- name: Create RDS cluster param group with no parameters set
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+    param_group_family: aurora5.6
+    description: test cluster param group
+  register: rds_cluster_param_group
+
+# Create RDS cluster param group with parameters
+- name: Create RDS cluster param group with no parameters set
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+    param_group_family: aurora5.6
+    description: test cluster param group
+    params: 
+      - ParameterName: binlog_format
+        ParameterValue: 'STATEMENT'
+        ApplyMethod: 'pending-reboot'
+
+      - ParameterName: binlog_checksum
+        ParameterValue: 'CRC32'
+        ApplyMethod: 'pending-reboot'
+  register: rds_cluster_param_group
+
+# Delete RDS cluster param group
+- name: Delete RDS cluster param group
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+  register: deleted_rds_cluster_param_group
+'''
+
+RETURN = '''
+DBClusterParameterGroups:
```

Comment:
Is it worth snake_casing the return results to be more ansiblish (not sure if snake_case copes with DBCluster or converts it to d_b_cluster!)